lib/perl/Genome/InstrumentData/AlignmentResult.pm

```diff
@@ -1146,6 +1146,15 @@ sub create_bam_header {
     return 1;
 }
 
+sub set_bam_size {
```

Comment:
Pushed a commit to change line 1073 to call `set_bam_size` and passing in `$bam`.